core/src/main/scala/kafka/server/BrokerMetadataCheckpoint.scala

```diff
@@ -96,11 +98,13 @@ case class LegacyMetaProperties(brokerId: Int,
   }
 }
 
-case class MetaProperties(clusterId: Uuid) {
+case class MetaProperties(clusterId: Uuid, brokerId: Int, controllerId: Int) {
   def toProperties(): Properties = {
     val properties = new Properties()
     properties.setProperty("version", 1.toString)
     properties.setProperty("cluster.id", clusterId.toString)
+    properties.setProperty(KafkaConfig.BrokerIdProp, brokerId.toString)
```

Comment:
please don't use config names here.  we might change them eventually

Reply:
Ack.